README.md

```diff
@@ -69,8 +69,8 @@ Linux, Apple OSX (Intel and M1)
 
 ## Getting started
 
-See the [DataFusion Comet User Guide](https://datafusion.apache.org/comet/user-guide/) for installation instructions.
+See the [DataFusion Comet User Guide](https://datafusion.apache.org/comet/user-guide/installation.html) for installation instructions.
 
 ## Contributing
 See the [DataFusion Comet Contribution Guide](https://datafusion.apache.org/comet/contributor-guide/contributing.html)
-for information on how to get started contributing to the project.
\ No newline at end of file
+for information on how to get started contributing to the project.
```

Comment:
What the difference on this sentence? Seems it is same but it is marked as in diffs.

Reply:
I think it's a new line at the end of the file.
I edited this from the GitHub UI.